BrainPortal/app/helpers/scope_helper.rb

```diff
@@ -681,13 +681,10 @@ def self.model_filter_values(model, attribute, label, association: nil)
     # Fetch the main filter values as an array of arrays:
     # [[value, label, count], [...]]
     model
-      .where("#{attribute} IS NOT NULL")
       .order(label, attribute)
       .group(attribute, label)
       .raw_rows(attribute, "#{label} AS #{label_alias}", "COUNT(#{attribute})")
       .reject { |r| r.first.blank? }
-      .map(&:to_a)
-      .to_a
```

Comment:
I'm pretty sure I didn't look up `raw_rows`' docs and just assumed Enumerable when I wrote this part. But if `raw_rows` does it, you're right that these two lines serve no purpose.